modules/ROOT/pages/api-gateway-encryption-mule3.adoc

```diff
@@ -0,0 +1,114 @@
+= Gateway Startup Encryption in Mule 3
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Gateway startup encryption allows you to encrypt sensitive information stored by the runtime such as policies, contracts, and the required credentials to configure the connection between your Mule application and Anypoint Platform.
+
+== Configuring the Runtime to Use Encryption
+
+You can encrypt your Anypoint Platform credentials (client ID, client secret, and proxy password) using a 16-characters long encryption key, and pass this encryption key to your runtime during start-up through the `anypoint.platform.encryption_key` property so that the runtime can decrypt and use your encrypted credentials:
+
+[source]
+----
+$MULE_HOME/bin/mule start -Danypoint.platform.encryption_key=MyEncryptionKey1
+----
+
+While the runtime is starting it checks for an encryption key. Then it gets the client ID and client secret values from its `wrapper.conf` file.
+
+== Encrypting Credentials using the Agent
+
+To configure the runtime through the agent to use encryption the argument `encryption-key` with the desired key as value. The agent then encrypts the client ID and Secret.
```

Comment:
There's something missing in this sentence: "To configure the runtime through the agent to use encryption the argument `encryption-key` with the desired key as value." 
Please check if this is the intended meaning: 
"To configure the runtime through the agent to use encryption, the argument `encryption-key` **must be configured** with the desired key as value.

Moreover, you could separate the runtime and the agent configuration to clarify: 
"To configure the runtime to use encryption, set the argument `encryption-key` with the desired key as value in the agent." or something like that.